.gitignore

```diff
@@ -17,3 +17,4 @@ docs/venv
 *.ipynb
 .vscode/
 read_Intan_RHD2000_file.m
+npy-matlab
```

Comment:
`npy-matlab` is a submodule, so if you've got this sitting in the repo folder you can just do

```shell
cd /path/to/JRCLUST
git submodule update --init --recursive
```
([source](https://stackoverflow.com/a/33795927/993881)).

That way you'll also get `matnwb`!

Reply:
Interesting. I aware of submodules until recently and still not very familiar with those. I'll try this out. I just don't want two instances of npy-matlab sitting in the path.

Reply:
Yeah, I can confirm that it will just add a .git folder and only complain at you if you've made changes locally, something like

![image](https://user-images.githubusercontent.com/844464/76020012-aca6dc80-5ef0-11ea-9d2b-7524e717c851.png)

Reply:
OK, cool. I think it would be worth adding a mention to that in the documentation.